src/versions.js

```diff
@@ -1,5 +1,5 @@
 const versions = {
-  emscripten: "3.1.11",
-  mfem: "v4.4-775-g80033b00f",
-  glvis: "v4.2",
+  emscripten: "3.1.51",
+  mfem: "v4.7-790-g3d43371e9",
```

Comment:
```suggestion
  mfem: "v4.7",
```
Like this?

Reply:
This build was done using `master` for mfem and glvis - are you suggesting to 1) just change what is shown in `versions.js` or to 2) change the build to use mfem `v4.7` rather than `master`?

Re 1): I think including the commit hash is very helpful for tracking which commit upstream broke functionality downstream.

Re 2): I think there is enough bug-fixes/changes between `mfem@v4.7` and `mfem@:master` to justify using `mfem@master`. I'm fine with building/tagging against `glvis@v4.3.1` but don't know if/when we'll have a `mfem@v4.7.1`.

We can always re-build/re-release when tagged versions of `mfem` and/or `glvis` are released but I think if we limit builds to that requirement it will take a while to propagate changes downstream (unless we start tagging upstream repos more frequently) :)

Reply:
I am just following the rule that releases depend on releases, so GLVis 4.3 depends on MFEM 4.7 and the same should be here that the new release of GLVis-js depends on GLVis 4.3.2 (the bugfix release) and MFEM 4.7. To be sure, the build should be also for MFEM 4.7 instead of master. Bugfix release of MFEM is not planned, as far as I know. If something is needed beyond 4.7 it means that a bugfix release would be needed there as well, but I hope not.

Reply:
Okay, that's fair. When we release `glvis@4.3.2` I'll re-build using that version + `mfem@v4.7` and update the build here.

Reply:
So what is the point of the build in this PR? I thought it is a release candidate 🤪 If it is not, why to update the build? And why to overstep the MFEM release to which we will be returning?

Reply:
okay, we can make a new PR then?

Reply:
I don't really understand your point, I'm agreeing with you

Reply:
If this PR is for testing (/release candidate), I would suggest to update the build to MFEM 4.7 and the latest version of GLVis, so we will keep linearly moving towards the release. Otherwise, I would drop the build update here and make a separate PR for the release (candidate) 😉 .

Reply:
```
  emscripten: "3.1.51",
  mfem: "v4.7",
  glvis: "v4.3-81-g7053ae2",
  ```

Reply:
Perfect 👍